frontend/src/services/settings.ts

```diff
@@ -53,6 +54,10 @@ export const maybeMigrateSettings = () => {
   if (currentVersion < 3) {
     localStorage.removeItem("token");
   }
+
+  if (currentVersion < 4) {
+    logout();
```

Comment:
is logout defined here? I don't see it imported...

Reply:
It's passed as a function param